### PR TITLE
verify autograph signatures, including autograph stage

### PR DESCRIPTION
**Pull request by escapewindow**
@g-k , is this something you feel comfortable reviewing?

### CODE REVIEW

**Comment by g-k** [comment]
@escapewindow sure

File: signingscript/sign.py [hunk 1]
```diff
@@ -736,6 +749,54 @@ def _owner_filter(tarinfo_obj):
...
+def verify_mar_signature(cert_type, fmt, mar):
+    """Verify a mar signature, via mardor.
+
+    Args:
+        cert_type (str): the cert scope string
+        fmt (str): the signing format
+        mar (str): the path to the mar file
+
+    Raises:
+        SigningScriptError: if the signature doesn't verify
```
Comment (g-k) [inline signingscript/sign.py]: nit: or the nick isn't found

File: signingscript/test/test_sign.py [hunk 2]
```diff
@@ -206,19 +211,22 @@ def test_build_signtool_cmd(context, signtool, from_, to, fmt):
     mocker.patch('signingscript.sign.requests.Session', Session_mock, create=True)
 
     context.task = {
-        'scopes': ['project:releng:signing:cert:dep-signing', 'project:releng:signing:format:autograph_mar']
+        'scopes': ['project:releng:signing:cert:dep-signing', 'project:releng:signing:format:{}'.format(format)]
```
Comment (g-k) [inline signingscript/test/test_sign.py]: :+1:

File: signingscript/sign.py (hunk 1 above)
Comment (escapewindow) [reply signingscript/sign.py]: Thanks!

**Pull request merged**